release/Magarena/scripts/Aetherworks_Marvel.groovy

```diff
@@ -2,8 +2,9 @@ def action = {
     final MagicGame game, final MagicEvent event ->
     final MagicCardList topCards = new MagicCardList(event.getRefCardList());
     event.processChosenCards(game, {
-        topCards.removeCard(it);
+        game.doAction(new RevealAction(it));
```

Comment:
RevealAction here is too late and not necessary, the card rearrangement occurs before the choice make for the event. For this card, the LookAction suffice to tell the AI that the cards are no longer hidden.